womtool/src/main/scala/womtool/cmdline/PartialWomtoolCommandLineArguments.scala

```diff
@@ -24,12 +25,13 @@ sealed trait WomtoolCommand
 
 object WomtoolCommand {
   case object Parse extends WomtoolCommand
-  case object Validate extends WomtoolCommand
   case object Highlight extends WomtoolCommand
   case object Inputs extends WomtoolCommand
   case object Graph extends WomtoolCommand
   case object Upgrade extends WomtoolCommand
   case object WomGraph extends WomtoolCommand
+
+  case class Validate(listDependencies: Boolean = false) extends WomtoolCommand
```

Comment:
Hmm... this doesn't appear to match the patter than the other command line options are using (they're all `case object` even though many of them also have their own parameters).

Not saying this is wrong necessarily, but it smells a little funny to me...

Reply:
I refactored it to match the other commands